packages/babel-core/test/fixtures/transformation/source-maps/input-source-map-multiple-output-sources-complete-replace/source-map.json

```diff
@@ -1,13 +1,15 @@
 {
-  "mappings": "AAAC;ACAD,K",
+  "version": 3,
+  "mappings": "AAAC,KCAD;ACAA,K",
```

Comment:
This is actually a bug fix. We were testing:

```js
if (
  this._lastGenLine === generatedLine &&
  this._lastSourceLine === line &&
  this._lastSourceColumn === column
) {
  return;
}
```

However, we forgot to check whether the filename is different! Eg, here were on the same generated line, and pointing at the same source line/column. But we're pointing at two totally separate source files. `AAAC ` is pointing at the firs file, and `KCAD` is pointing at the second file.